index.css

```diff
@@ -0,0 +1,237 @@
+  width: 100%;
+  max-width: 1440px;
+  margin: auto;
+  font-family: "Lato", sans-serif;
+}
+
+.logo {
+  min-height: 80vh;
+  height: 36px;
+  background-image: url("images/background.jpg");
+  background-repeat: no-repeat;
+  background-position: 50% 100%;
+  background-size: cover;
+  padding-top: 18vh;
+}
+
+.flex {
+  display: flex;
+}
+
+.nav {
+  width: 850px;
+  height: 36px;
+  margin: 0 auto;
+  align-items: center;
+  justify-content: center;
+  border-radius: 20px;
+  padding: 0 10px;
+  position: relative;
+  margin-top: -30px;
+  background: rgb(144, 144, 144);
+  background:
+    linear-gradient(
+      180deg,
+      rgba(144, 144, 144, 1) 0%,
+      rgba(51, 51, 51, 1) 50%,
+      rgba(133, 133, 133, 1) 82%
+    );
+}
+
+.fa.fa-search {
+  color: yellowgreen;
+  padding-top: 5px;
+}
+
+.links i {
+  font-size: 20px;
+  color: white;
+}
+
+.text {
+  text-align: center;
+}
+
+.links {
+  color: #fff;
+  line-height: 2;
+  font-size: 14px;
+  flex: 1;
+  background: rgb(144, 144, 144);
+  background:
+    linear-gradient(
+      180deg,
+      rgba(144, 144, 144, 1) 0%,
+      rgba(51, 51, 51, 1) 50%,
+      rgba(133, 133, 133, 1) 82%
+    );
+  border-right: solid thin rgba(255, 255, 255, 0.15);
+  padding: 0 10px;
+}
+
+.links:first-child {
+  padding-top: 5px;
+}
+
+.links:last-child {
+  border: none;
+}
+
+.links a {
+  color: #fff;
+  height: 32px;
+}
+
+.gallery a {
+  width: 25%;
+}
+
+.section-one a,
+.section-two a,
+.section-three a {
+  color: blue;
+}
+
+.links a:hover {
+  text-decoration: none;
+}
+
+.item:hover {
+  cursor: pointer;
+  background: rgb(103, 102, 102);
+  background:
+    linear-gradient(
+      180deg,
+      rgba(103, 102, 102, 1) 0%,
+      rgba(51, 51, 51, 1) 50%,
+      rgba(88, 88, 88, 1) 82%
+    );
+}
+
+form {
+  position: relative;
+}
+
+form span {
+  position: absolute;
+  left: 8px;
+}
+
+form input {
+  padding: 1px;
+  padding-left: 20px;
+  color: #333;
+  background: transparent;
+  transition: 0.5s;
+  border-radius: 14px;
+  width: 100px;
+}
+
+form input:focus {
+  outline: none;
+  background-color: #fff;
+  width: 120px;
+}
+
+form input:focus + span {
+  color: #333;
+}
+
+.fa.fa-play-circle {
+  font-size: 40px;
+}
+
+.set {
+  padding-top: 40px;
+}
+
+.white {
+  color: #fff;
+}
+
+h3 {
+  margin: 20px auto;
+}
+
+.gallery img {
+  border: 1px solid #fff;
+}
+
+.gallery {
+  width: 100%;
+  flex: 1;
+  justify-content: space-around;
+  overflow: hidden;
+  padding: 0 0 40px;
+}
+
+/* footer section */
+footer {
+  margin-top: 100px;
+}
+
+.section-one,
+.section-two,
+.section-three {
+  width: 50%;
+  float: left;
+  margin-top: -1px;
+  min-height: 10px;
+  font-size: 12px;
+}
+
+nav ul li,
+footer ul li {
+  text-align: center;
+  padding: 0 5px;
+  display: inline;
+  border-right: solid thin #333;
+}
+
+p {
+  margin-top: 4px;
+}
+
+.section-one p {
+  font-size: 10px;
+  text-align: center;
+}
+
+nav ul li:last-child,
+footer ul li:last-child {
+  border: none;
+}
+
```

Comment:
## Using Inefficient CSS Selectors[1]

When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

This can be avoided by not using IDs within the selectors as they are directly specific.

 - [ ]  Please add a class to `<ul>` element` and delete the left unnecessary rules

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

Reply:
I'm using a class for the ul?